### ISSUE TITLE
[5.7.0] SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by hswong3i** [ansible-community/ansible-build-data#114]
When packing Ansible 5.7.0 as .deb/.rpm (see https://github.com/alvistack/ansible-community-ansible-build-data/commit/f3cb88964b44be336e8d543309c80b4d7d442d50), apt generate following error message when install the result package:
```
Setting up ansible (100:5.7.0-1) ...
  File "/usr/lib/python3/dist-packages/ansible_collections/fortinet/fortios/plugins/modules/fortios_wirel
ess_controller_hotspot20_h2qp_advice_of_charge.py", line 281
    filtered_data =
                  ^
SyntaxError: invalid syntax

  File "/usr/lib/python3/dist-packages/ansible_collections/fortinet/fortios/plugins/modules/fortios_wirel
ess_controller_hotspot20_h2qp_osu_provider_nai.py", line 238
    filtered_data =
                  ^
SyntaxError: invalid syntax

  File "/usr/lib/python3/dist-packages/ansible_collections/fortinet/fortios/plugins/modules/fortios_wirel
ess_controller_hotspot20_h2qp_terms_and_conditions.py", line 236
    filtered_data =
                  ^
SyntaxError: invalid syntax

dpkg: error processing package ansible (--configure):
 installed ansible package post-installation script subprocess returned error exit status 1
```

See https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/pull/177

**Comment by gotmax23** [comment]
Is it possible to cut a hot-fix release?

**Comment by felixfontein** [comment]
+1 for a new release with the version of that collection restricted to the version in the previous release. Especially because we still had no reaction by the collection maintainers.

**Comment by hswong3i** [comment]
The hotfix is relatively simple (https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/pull/177#issuecomment-1111601143):
```
pushd ansible_collections/fortinet/fortios
curl -skL https://patch-diff.githubusercontent.com/raw/fortinet-ansible-dev/ansible-galaxy-fortios-collection/pull/177.patch | patch -p1
popd
```

And my hotfix package already successfully build (https://build.opensuse.org/package/show/home:alvistack/ansible-community-ansible-build-data-5.7.0) since 4 days ago...

**Comment by gotmax23** [comment]
Yeah, I did something similar in the Fedora package. If feasable, it would probably be better to patch the current version as opposed to downgrading it.

**Comment by hswong3i** [comment]
> Yeah, I did something similar in the Fedora package. If feasable, it would probably be better to patch the current version as opposed to downgrading it.

+1, keep version constraint (as already **carelessly** released) and patch it ;-)

**Comment by felixfontein** [comment]
We do not modify collection content in Ansible releases, so either the maintainers fix it (nothing happened so far), or we downgrade to the previous working version of that collection.

**Comment by hswong3i** [comment]
The fixes itself is just a simple technical issue about remove 3 enters...

But the hotfix for such careless release is now blocked by political issue 🧐

**Comment by dmsimard** [comment]
> The fixes itself is just a simple technical issue about remove 3 enters...
> 
> But the hotfix for such careless release is now blocked by political issue monocle_face

First, thanks for filing this issue here and in the collection repository -- I see that a fix has been merged but there hasn't been a new version of the collection released yet.

To be clear, there's no politics involved -- we are not the owners and maintainers of every collection that ends up in the package and it is community-driven.

We are working on improving our test coverage to prevent an issue like this one from happening again but in the meantime we will build and release an ansible 5.7.1 today with a previous version of the fortinet.fortios collection.

**Comment by hswong3i** [comment]
Well my unofficial OBS build for 5.7.1 looks good enough without additional hotfix patch:
* https://build.opensuse.org/package/show/home:alvistack/ansible-community-ansible-build-data-5.7.1
* https://github.com/alvistack/ansible-community-ansible-build-data/commit/58ec392b01d81bb29598adebc85982d53485daeb

@dmsimard thank you very much ;-)
